### PR TITLE
added customer voicebot

### PR DESCRIPTION
**Pull request by rbgo404**
Please check this PR

### CODE REVIEW

File: cookbook/serverless-customer-service-bot.mdx [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+__Total Billable Hours with Inferless:__ 0.8 (inference duration) + 1.59 (idle time) + 0.67 (cold start overhead)  = 3.06 hours<br/>
+__Total Billable Hours with Inferless:__ 3.06 hours<br/>
+
+
+| Scenarios | On-Demand Cost | Serverless Cost|
+| :--- | :---- | :---- |
+|  100 requests/day | \$28.8 (24 hours billed at $1.22/hour) | \$2.85 (2.34 hours billed at $1.22/hour) |
+|  1000 requests/day | \$28.8 (24 hours billed at $1.22/hour) | \$3.73 (3.06 hours billed at $1.22/hour) |
+
+By opting for Inferless, **_you can achieve up to 163.98% cost savings._**<br/>
```
Comment (AGInfer) [inline cookbook/serverless-customer-service-bot.mdx:290]: @rbgo404 % looks wrong here

**Review by AGInfer — COMMENTED** [review]
290 row % looks wrong

File: cookbook/serverless-customer-service-bot.mdx (hunk 1 above)
Comment (rbgo404) [reply cookbook/serverless-customer-service-bot.mdx:290]: I am taking the smallest number i.e 2.85. And comparing with 28.8.
Here's a SS from calculator:
image

Comment (AGInfer) [reply cookbook/serverless-customer-service-bot.mdx:290]: wrong formula rajdeep..

**Comment by rbgo404** [comment]
I have updated, please check now

**Pull request merged**